features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -101,14 +124,57 @@ The PSA on single v7-M is supported with the help of Mbed OS PSA services. The f
     }
 ```
 
-Please pay attention to config options `extra_labels_add` and `device_has_add`. If needed then a PSA target definition **MUST** use [extra_labels/device_has]`_add` or [extra_labels/device_has]`_remove` (not `extra_labels` or `device_has`) to add/remove either extra_labels or target capabilities. Also, use `[feature_]`add/remove to add/remove a feature.
+Please pay attention to config options `extra_labels_add` and `device_has_add`. If needed then a PSA target definition **MUST** use [extra_labels/device_has]`_add` or [extra_labels/device_has]`_remove` (not `extra_labels` or `device_has`) to add/remove either extra_labels or target capabilities. Also, use `feature_`[add/remove] to add/remove a feature.
 Check [extra_labels](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html), [device_has](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) and [features](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) for more information.
 
-### Adding dual v7-M targets
+## Adding dual-core PSA targets
+A target can be categorized as a dual-core target if it has at least two cores that are either Armv7-M or Armv6-M. On dual-core PSA targets, TF-M runs on the SPE and provides PSA services. Therefore, it is **MANDATORY** to define both the SPE and the NSPE target for dual-core PSA targets.
+
+The SPE target **MUST** contain the following attributes:
+
+* `inherits`: PSA generic target PSA_DUAL_V7_M_SPE
+* `tfm_target_name`: Target name in TF-M
+* `tfm_bootloader_supported`: If TF-M bootloader is supported by the target. Values supported are "true" and "false"
+* `tfm_default_toolchain`: Default TF-M toolchain supported. Values supported are `ARMCLANG` and `GNUARM`
```

Comment:
Do we need to change the names of the compilers from standard Mbed ones?

Reply:
@bulislaw Unfortunately yes. TF-M uses different naming conventions than Mbed OS. We are using TF-M build system to generate the binary. Therefore we have to use the naming followed by TF-M. Since this is a target specific parameter, it is added to `targets.json`.